api/InputEvent.json

```diff
@@ -206,10 +206,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "83"
```

Comment:
Confirmed by running http://mdn-bcd-collector.appspot.com/tests/api/InputEvent/getTargetRanges in Firefox 82 and 83.